app/assets/javascripts/visualizations/highvis/disabledVis.coffee

```diff
@@ -39,7 +39,8 @@ $ ->
           <img src='#{window.icons["novis_timeline"]}'>
           <br>
           <br>
-          Either a time field was not found or there were not enough data
+          Either Timestamp field or Number field is missing from project or
+          there is not enough data. (min of 3 points required)
```

Comment:
This just sounds awkward.  How about:
"Either a timestamp or number field is missing, or there is not at least 3 data points."